### PR TITLE
Improve messages from Bandwidth update code to be programmatically parsed

### PR DESCRIPTION
**Pull request by nakulkar-msft**
1. The messages from console are handled by watchInputs() routine on LCM.
2. The message is forwarded to JohsAdmin, and watchInputs() will wait till action is completed.

### CODE REVIEW

File: common/messages.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	TimeStamp time.Time       `json:"TimeStamp"`
+	MsgType   string          `json:"ResponseType"`
+	Value     fmt.Stringer    `json:"Value"`
+	Err       error           `json:"-"`
+}
+
+type LCMMsg struct {
+	Req      *LCMMsgReq
+	Resp     *LCMMsgResp
+	respChan chan bool
```
Comment (zezha-msft) [inline common/messages.go:50]: couldn't the respChan give back Resp directly instead of a bool?

File: jobsAdmin/JobsAdmin.go [hunk 2]
```diff
@@ -508,53 +508,62 @@ func (ja *jobsAdmin) TryGetPerformanceAdvice(bytesInJob uint64, filesInJob uint3
...
-}
-
-func (ja *jobsAdmin) messageHandler(inputChan <-chan common.LCMMsg) {
+func (ja *jobsAdmin) messageHandler(inputChan <-chan *common.LCMMsg) {
 	toBitsPerSec := func(megaBitsPerSec int64) int64 {
 		return megaBitsPerSec * 1000 * 1000 / 8
 	}
-	glcm := common.GetLifecycleMgr()
 	
 	lastPerfAdjustTime := time.Now() // right when this routine songs
```
Comment (zezha-msft) [inline jobsAdmin/JobsAdmin.go:516]: Consider adjusting lastPerfAdjustTime to time.Now - 2 * minIntervalBetweenPerfAdjustment or something like that

**Review by zezha-msft — APPROVED** [review]
lgtm

could you please adjust the default value of lastPerfAdjustTime so that the cx can give an adjustment right when the command starts? This was a point of confusion as the first adjustment came in too soon and they got an error back.

**Review by siminsavani-msft — APPROVED** [review]

**Comment by zezha-msft** [comment]
@nakulkar-msft lgtm, seems like there's a flaky test, so I retriggered the CI. Feel free to merge once build is green

**Pull request merged**